tests/runner.py

```diff
@@ -213,6 +213,9 @@ def setUp(self):
         if temp_file.endswith('.ll'):
           self.has_prev_ll = True
 
+    # Cache check results
```

Comment:
Doing this for each test is more than we need, I think. We can just do it once at the very beginning, at the top level?

Reply:
Oops, I probably put it in the wrong place. I thought to place it to the global setup function.